tests/cluster/cluster.go

```diff
@@ -41,50 +38,58 @@ import (
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/tools/remotecommand"
+
+	test "github.com/onsi/ginkgo"
+	matcher "github.com/onsi/gomega"
 )
 
-type ClusterApi struct {
+// Cluster object containing k8s cni cncf api client referneces
+type Cluster struct {
```

Comment:
Why are we renaming this? Should not it become ClusterAPI instead?

Reply:
Yes, of course, changed it into ClusterAPI.